amqp.c

```diff
@@ -395,6 +395,10 @@ PHP_INI_BEGIN()
 	PHP_INI_ENTRY("amqp.channel_max",		DEFAULT_CHANNEL_MAX,		PHP_INI_ALL, NULL)
 	PHP_INI_ENTRY("amqp.frame_max",			DEFAULT_FRAME_MAX,			PHP_INI_ALL, NULL)
 	PHP_INI_ENTRY("amqp.heartbeat",			DEFAULT_HEARTBEAT,			PHP_INI_ALL, NULL)
+	PHP_INI_ENTRY("amqp.cacert",			DEFAULT_CACERT,				PHP_INI_ALL, NULL)
+	PHP_INI_ENTRY("amqp.cert",				DEFAULT_CERT,				PHP_INI_ALL, NULL)
+	PHP_INI_ENTRY("amqp.key",				DEFAULT_KEY,				PHP_INI_ALL, NULL)
+	PHP_INI_ENTRY("amqp.verify",			DEFAULT_VERIFY,				PHP_INI_ALL, NULL)
```

Comment:
Yes. Before rabbitmq-c 0.8.0 there was no way to verify them separately (see [this changes below](https://github.com/pdezwart/php-amqp/pull/205/files#diff-89309d5b78d7c1a44970257f5ec91eb6R325)), so I make single general option to not bother users with rabbitmq-c API changes. Anyway, in prod all verifications should be turned on.